index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile first</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="header1">
+
+        <h3 class="logo">My Logo</h3>
+        <img src="./img/hamburger.png" alt="hamburger_image">
+
+    </div>
+
+    <section class="section1">
+
+        <div class="intro">
+
+            <h1>I'm Emeka Glad to see you!</h1>
+
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! if you like what you see and have a project you need coded, don't hesistate to contact
+                me.
+            </p>
+
+        </div>
+
+        <div class="footer1">
+
+            <p>LET'S CONNECT</p>
+
+            <figure class="footer_figure">
+                <img src="./img/socialmedia.png" alt="contact_image">
+            </figure>
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I appreciate what you did, but your list of `icons` should use a `ul` tag to group them, and `li` per icon; each icon should has a link (`a` tag) 😄